src/internal/xjwt_keyset.h

```diff
@@ -25,11 +25,11 @@ extern "C" {
 #endif /* __cplusplus */
 
 struct xjwt_keyset_t {
-  xjwt_key_t* keys;
+  xjwt_key_t *keys;
```

Comment:
did clang format do this?

Reply:
Yeah, I can make my editor match the format command I think, one sec.

Reply:
Oh:

clang-format figures it out from other files I think, unless we do something like: 

```
DerivePointerAlignment: false
PointerAlignment: Left
```

Do you want to pick one and reformat everything?

Reply:
there's a scons target with the clang-format settings, `scons format`, i guess we should just check-in `.clang-format` file.